kodilla-good-patterns/src/main/java/com/kodilla/kodillagoodpatterns/Food2Door/FoodOrderService.java

```diff
@@ -0,0 +1,11 @@
+package com.kodilla.kodillagoodpatterns.Food2Door;
+
+public class FoodOrderService implements SendingService {
+
+    public boolean send(final Company company, String productType, int quantity){
+        System.out.println("Preparing order from " + company.getCompanyName()
+                + " with " + productType + " in an amount of " + quantity);
+
+        return true;
```

Comment:
to powinien byc producent jedzenie. Kazdy z 
```
 ExtraFoodShop, HealthyShop, GlutenFreeShop
 ```
 powinien miec swoja implementacje